### PR TITLE
Update/read me

### LINKED ISSUE #27: Update Read Me.
We can do this towards the last day. Just wanted to create the issue as its one of our ToDos. I made myself a template I use for every project for my Read Me files. Can do something similar, shouldn't take to long.

- [ ] Do a nice write up for our Read Me file

### PR DESCRIPTION
**Pull request by timjacksonm**
Template started
Summary still needs to be written.
A gif still needs to be created. <-- I can do this just waiting for final version of game. 
If you guys want more social links listed in the read-me let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,113 @@
...
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+**How to Play?**
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+<p>The player will use the dinosaur legend on each side of the battlefield to understand the point value of a specific dinosaur.
+We encourage the player to group like dinosaurs together when adding up the totals. Using your imagination with addition or multiplication will make adding their values easier.</p>
+   
+Be careful! The game will add in subtraction dinosaurs as the difficulty increases.
+   
+<p>The player will adjust the input field to equal each teams total point value.
```
Comment (savwiley) [inline README.md]: The same typos from the rules modal are here, too. Just to point them out real fast:

Ln. 62. `each team's total`
Ln. 63. `of your answer, click Battle!`
Ln. 69. `each dinosaur's value`
Ln. 70. `each team's total`
Ln. 71. `If at least one`

File: README.md [hunk 2]
```diff
@@ -1,70 +1,113 @@
...
+## Summary
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+To be written.
 
-### Deployment
+## Authors
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+| 👤 **Tim Jackson**                                           | 👤 **savwiley**                                 | 👤 **rankoliang**                                   |
```
Comment (savwiley) [inline README.md]: Ah, this is nice! This isn't part of the review, but my twitter is `https://twitter.com/savwiley` and my linkedin is `https://www.linkedin.com/in/savwiley`.

File: README.md (hunk 1 above)
Comment (timjacksonm) [reply README.md]: Thank you! Updated.

File: README.md (hunk 2 above)
Comment (timjacksonm) [reply README.md]: added.

**Review by rankoliang — CHANGES_REQUESTED** [review]
A summary could be

```
This is a counting game where you count dinosaurs and watch them battle! This game was made for [The Odin Project's first game jam](https://itch.io/jam/top-jam-1), where the theme was edutainment.
```
I would move this near the top.

I would add in some technical details on how to run our app too. 
```md
### Development
Clone this project and run in the project's directory
```yarn install```
to install all of the dependencies.

To run the development server, run the command
```yarn start```
and visit [http://localhost:3000](http://localhost:3000).
```

Looks good otherwise!

**Comment by timjacksonm** [comment]
Re submitted new PR without all the extra commits. #36

**Pull request closed without merging**